PurchasesCoreSwift/Networking/HTTPClient.swift

```diff
@@ -316,30 +319,4 @@ private extension HTTPClient {
         }
         return urlRequest
     }
-
-    func assertIsValidRequest(httpMethod: String, requestBody: [String: Any]?) throws {
```

Comment:
I've removed this assertion since the HTTP method is not passed as param. Do you think we should continue checking this?

Reply:
I think that as long as the public methods don't allow for the wrong combination of parameters to be sent in, there's no need to perform further assertions.